moveit_ros/planning/trajectory_execution_manager/src/trajectory_execution_manager.cpp

```diff
@@ -111,6 +110,15 @@ void TrajectoryExecutionManager::initialize()
   execution_duration_monitoring_ = true;
   execution_velocity_scaling_ = 1.0;
 
+  if (!node_handle_.getParam("allowed_execution_duration_scaling", allowed_execution_duration_scaling_))
+    allowed_execution_duration_scaling_ = DEFAULT_CONTROLLER_GOAL_DURATION_SCALING;
+
+  if (!node_handle_.getParam("allowed_goal_duration_margin", allowed_goal_duration_margin_))
+    allowed_goal_duration_margin_ = DEFAULT_CONTROLLER_GOAL_DURATION_MARGIN;
+
+  if (!node_handle_.getParam("allowed_start_deviation", allowed_start_deviation_))
```

Comment:
can you add an example of this setting (and perhaps the other two above) to the setup_assistant templates within this PR?